packages/iov-bns-governance/src/governor.spec.ts

```diff
@@ -23,7 +23,7 @@ function pendingWithoutBnsd(): void {
 const adminMnemonic = "degree tackle suggest window test behind mesh extra cover prepare oak script";
 const adminPath = HdPaths.iov(0);
 const bnsdUrl = "ws://localhost:23456";
-const guaranteeFundEscrowId = Encoding.fromHex("88008800");
+const guaranteeFundEscrowId = 2281736192;
```

Comment:
Is this even used anywhere? Looks like a very very big number given this is a counter

Reply:
It's used in the test for `ReleaseGuaranteeFunds`. I can change it to a more plausible number.